packages/design-system/.storybook/docs/tokensDocHelpers/components/DefinitionList/DefinitionListItem/DefinitionListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from 'react';
+import { Token } from '../../../../../../src/tokens/types';
+import styles from './DefinitionListItem.module.scss';
+import TokenName from '../../../TokenName';
+import { getCssName, getScssName } from '../../../TokenFormatter';
+import CopyValue from './CopyValue';
+import classnames from 'classnames';
+
+function DefinitionListItem({ token, children }: { token: Token; children?: ReactElement }) {
+	if (!token) {
+		return null;
+	}
+
+	return (
+		<div className={classnames(styles.listItem, { [styles.listItem__noCard]: !children })}>
+			{children && <dd className={styles.listItem__demo}>{children}</dd>}
```

Comment:
Not sure it's valid html.
`dd` must follow a `dt` or another `dd` (in case of multiple definition for the same term)

Reply:
I've checked (it's Fabien's code) and it turns out it _is_ valid!

Reply:
Weird, an online checker told me it was not